docs/getting-started/integration-method/gateway-fallbacks.mdx

```diff
@@ -24,61 +42,81 @@ export type HeliconeFallback = {
-    print(response.status_code)
-    print(response.text)
-
-    ```
-
-  </Tab>
-  <Tab title="Other Languages">
-    For integration within other languages, please refer to our guide on [adding a Helicone Header](/helicone-headers/header-directory).
-  </Tab>
-
-</Tabs>
+### Extracting Fallbacks Response Header
+
+When fallbacks is enabled, you can capture the fallback index from the headers of the response returned.
+
+
+```ts
+Helicone-Fallback-Index: number // indicates the fallback  used.
+```
+
+### Example
+
+Using the OpenAI API and set LemonFox API as fallback, then extracting the header.    
+
+  <Tip>For integration with other languages, please refer to the format in [Header Directory](/helicone-headers/header-directory).</Tip>
+
+  ```python Python (w/o package)
+  url = "https://gateway.hconeai.com/v1/chat/completions"
+
+  headers = {
+    "Content-Type": "application/json",
+    "Helicone-Auth": "Bearer <Helicone KEY>",
+    "Helicone-Fallbacks": json.dumps([ # add this new header
+        {   # this API will be used if available, otherwise, the system will use the next one
+            "target-url": "https://api.openai.com",
+            "headers": {
+              "Authorization": "Bearer <OpenAI Key>",
+            },
+            "onCodes": [{"from": 400, "to": 500}]
+        },
+        {   # set fallback to LemonFox API
+            "target-url": "https://api.lemonfox.ai", 
+            "headers": {
+                "Authorization": "Bearer <LemonFox Key>",
+                "Content-Type": "application/json",
+            },
+            "onCodes": [401, 403],
+            "bodyKeyOverride": {
+                "model": "zephyr-chat",
+            }
+        },
+    ]),
+  }
+  payload = {
+    "model": "gpt-4",
+    "messages": [
+      {
+        "role": "user",
+        "content": "What is the meaning of life?"
+      }
+    ],
+    "max_tokens": 300
+  }
+
+  response = requests.post(url, headers=headers, data=json.dumps(payload))
+  print("Fallback index used: ", response.headers["helicone-fallback-index"])
+  print(response.status_code)
+  print(response.text)
+
+  ```
+
+
+## Limitations 
+
+### Using Approved Domains
+Helicone supports integration with several approved domains for providers such as OpenAI, Anthropic, Together-AI, OpenRouter, and Azure. 
+
+Choosing from our [latest list of approved domains](https://github.com/Helicone/helicone/blob/360221606ddcfb871920f7a54379cf82a5d883a5/costs/src/providers/mappings.ts#L15) as fallbacks means you can send **unlimited requests** to the providers via Helicone. 
+
+### Using Unapproved Domains
+You can also use unapproved domains as fallbacks. However, to protect our community from potential threats, we have certain restrictions for unapproved domains:
+- 10,000 requests per day
+- 1 request per second
+
+
+
+## Questions? 
+
+<Tip>Questions or feedback? Reach out to us at [help@helicone.ai](help@helicone.ai). </Tip>
```

Comment:
Can you add aa call link down here too?

Reply:
will make this a reusable snippet soon to include scheduling calls, and updated all docs at once!